sig/handwritten/app/models/user.rbs

```diff
@@ -8,7 +8,9 @@ class User < ApplicationRecord
 
   # User has profile and authentication_provider_github at least one
   def profile: () -> Profile
+             | ...
```

Comment:
I added an overload statement (`| ...`) to avoid the conflicts with types from rbs_rails.